BrowserKit/Sources/ComponentLibrary/CollapsibleCardView.swift

```diff
@@ -167,24 +173,15 @@ public class CollapsibleCardView: ShadowCardView, UIGestureRecognizerDelegate {
 
         headerView.addSubview(titleLabel)
```

Comment:
hey @imsanchit There's a separate issue for CollapsibleCardView, could you move it in a separate PR? https://github.com/mozilla-mobile/firefox-ios/issues/16798

Reply:
Thanks @razvanlitianu for pointing this out, I have removed the code for CollapsibleCardView to separate PR.